R/vector-chr.R

```diff
@@ -82,7 +99,7 @@ string <- function(x, encoding = NULL) {
 #' as_bytes(latin1)
 #' }
 as_utf8_character <- function(x) {
-  enc2utf8(as_character(x))
+  .Call(rlang_unescape_character, as_character(x))
```

Comment:
Calling `as_utf8_character()` without having read its documentation and understood what it does. This can be us, a package that uses rlang, or user code, perhaps already implemented against rlang 0.1.1.